packages/@react-spectrum/card/docs/CardView.mdx

```diff
@@ -0,0 +1,509 @@
+{/* Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/card';
+import cardViewTypes from 'docs:@react-types/card/src/index.d.ts';
+import {HeaderInfo, PropTable, PageDescription, TypeLink, ClassAPI, FunctionAPI, InterfaceType, TypeContext} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/card/package.json';
+
+```jsx import
+import {Card, CardView, GalleryLayout, GridLayout, WaterfallLayout} from '@react-spectrum/card';
+import {Flex} from '@react-spectrum/layout';
+```
+
+---
+category: Collections
+keywords: [cardview, gallery, grid, waterfall, tile]
+---
+
+# CardView
+
+<PageDescription>{docs.exports.CardView.description}</PageDescription>
+
+<HeaderInfo
+  packageData={packageData}
+  componentNames={['Card', 'CardView', 'GalleryLayout', 'GridLayout', 'WaterfallLayout']} />
```

Comment:
```suggestion
  componentNames={['Card', 'CardView', 'GalleryLayout', 'GridLayout', 'WaterfallLayout']}
    sourceData={[
    {type: 'Spectrum', url: 'https://spectrum.adobe.com/page/cards/#Layout'}
  ]} />
```
Not sure if we'd want to include this or not.

Reply:
yeah, I wasn't sure as well. I'm leaning towards not including a Spectrum link until the up to date one from contributions makes it into the public site so that it shows the updated guidance on focusable elements within the cards

Reply:
Good idea, that makes more sense.